packaging/greenboot/microshift-greenboot-required.sh

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
```

Comment:
Yes, here's the logic in the naming of the script:
- Greenboot has "required" and "wanted" checks, so using "required" in the script name denotes mandatory checks
- For the "greenboot" in the name, the script is integrated in the Greenboot upgrade flow. However, it can be run in a standalone fashion and the checks will work.

We can rename the script to something more generic, but it's intended to be used with Greenboot. 
Any further suggestions for changing the name?

Reply:
Here are the names of the scripts in the optional `greenboot-default-health-checks` RPM:
```
/usr/lib/greenboot/check/required.d/01_repository_dns_check.sh
/usr/lib/greenboot/check/required.d/02_watchdog.sh
/usr/lib/greenboot/check/wanted.d/01_update_platforms_check.sh
```

I notice we may want to prefix a numbering to order this check after the base OS checks in case both are installed. Other than that, there doesn't seem to be a naming convention. I would have assumed names express the property being checked (e.g. "microshift_up").